sceptre/cli/init.py

```diff
@@ -71,11 +72,12 @@ def init_project(ctx, project_name):
         "region": os.environ.get("AWS_DEFAULT_REGION", "")
     }
 
-    config_path = os.path.join(cwd, project_name, "config")
-    _create_config_file(config_path, config_path, defaults)
+    config_path = os.path.join(context.project_path, project_name,
```

Comment:
`SceptreContext(..)...command_path_is_stack` possibly

Reply:
Not sure about this one?